docs/modules/servers/pages/postgres/configure/blobstore.adoc

```diff
@@ -0,0 +1,6 @@
+= Postgresql James Server &mdash; blobstore.properties
+:navtitle: blobstore.properties
+
+:sample-configuration-prefix-url: https://github.com/apache/james-project/blob/postgresql/server/apps/postgres-app/sample-configuration
+:pages-path: postgres
+include::partial$configure/blobstore.adoc[]
```

Comment:
I remember we have options for POSTGRES and FILE implementation too.

Reply:
Your mean is we should add more section for `docs/modules/servers/pages/postgres/configure/blobstore.adoc` ?

include::partial$configure/blobstore.adoc[] already contains the "file" and I have already added "postgres" option (not in detail, but it contains the option).